src/lib/containers/table/SynapseTable.tsx

```diff
@@ -80,18 +81,16 @@ type Info = {
   index: number
   name: string
 }
-export interface Dictionary<T> {
-  [key: string]: T
-}
```

Comment:
Dictionary<T> is the same thing as Record<string, T>, which is built-in, so I removed this.